### PR TITLE
change the toolkit version

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4562
https://tools.hmcts.net/jira/browse/EUI-4563

### Change description ###
Change the toolkit version

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/services/ccd-config/ccd-case.config.ts [hunk 1]
```diff
@@ -160,6 +160,10 @@ export class AppConfig extends AbstractAppConfig {
     return this.workallocationUrl;
   }
 
+  public getRefundsUrl(): string {
```
Comment (connorpgpmcelroy) [inline src/app/services/ccd-config/ccd-case.config.ts:163]: What is the purpose of this? Is this to be changed at some point?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Build not running and have query about getRefundsUrl()

File: src/app/services/ccd-config/ccd-case.config.ts (hunk 1 above)
Comment (kuda1992) [reply src/app/services/ccd-config/ccd-case.config.ts:163]: The new CCD config has a required field getRefundsUrl.

**Review by udaydenduluri33 — APPROVED** [review]
@kuda1992  thanks for the changes

**Pull request merged**